cfgov/v1/atomic_elements/molecules.py

```diff
@@ -71,6 +74,12 @@ class Meta:
 
 
 class TextIntroduction(blocks.StructBlock):
+    eyebrow = blocks.CharBlock(
+        required=False,
+        help_text=('Optional: Adds an H5 eyebrow above H1 heading text. '
+                   'Only use in conjunction with heading.'),
+        label="Pre-heading"
```

Comment:
Should this be single quotes?

Reply:
TRIVIAL: Also is space around `=` sign a style standard?

Reply:
Yes, quotes should be single quotes. No, weirdly, lack of space around the equals sign is the standard.

Reply:
@anselmbradford Single quotes added in https://github.com/cfpb/cfgov-refresh/pull/4004/commits/1cde5670e657cf6e4086a281703bd521d47cf9ea -- thanks for catching that!

For the spacing, I think the convention is that there is space around the equals sign except in keyword arguments.